### PR TITLE
Bumping dependencies

### PR DESCRIPTION
**Pull request by FK78**
Resolves multiple dependabot PRs.

**Overall change:** _Bumping dependencies._

**Code changes:**

- _Bump storybook-chromatic from 3.5.0 to 3.5.1 #3086_
- _Bump ramda from 0.26.1 to 0.27.0 #3087_
- _Bump @storybook/theming from 5.3.9 to 5.3.12 #3088_
- _Bump @testing-library/jest-dom from 5.0.2 to 5.1.1 #3089_
- _Bump @storybook/addon-a11y from 5.3.9 to 5.3.12 #3090_
- _Bump @storybook/addon-actions from 5.3.9 to 5.3.12 #3091_
- _Bump @storybook/addon-viewport from 5.3.9 to 5.3.12 #3092_
- _Bump @storybook/react from 5.3.9 to 5.3.12 #3093_
- _Bump stylelint from 13.0.0 to 13.1.0 #3094_
- _Bump eslint-plugin-import from 2.20.0 to 2.20.1 #3095_
- _Bump @storybook/addon-notes from 5.3.9 to 5.3.12 #3096_
- _Bump @storybook/addon-knobs from 5.3.9 to 5.3.12 #3097_
- _Bump package.json, package-lock.json and CHANGELOG.md from 2.0.73 to 2.0.74_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by simonsinclair — APPROVED** [review]

**Review by thekp — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+
```
Comment (thekp) [inline CHANGELOG.md]:
```suggestion
```

Comment (FK78) [reply CHANGELOG.md]: This was done to run `synk`, it has been reverted.

**Pull request merged**